### DISCUSSION TITLE
Can the window tiling be positioned on top of float windows?

### DISCUSSION THREAD

**Discussion by wastee** [hyprwm/Hyprland#2473] [Q&A]
I currently use "cyclenext" and "bringactivetotop" to switch between windows. However, when there are both float windows and tiled windows present, the float windows always remain on top, even when they are not in focus.

Is there any way to achieve the desired behavior?

**Comment by Nicknamely** [comment]
Did you find your answer ?

**Comment by johnwill97** [comment]
I've read and tried the directions from the [wiki](https://wiki.hyprland.org/), but it seems that tiling window to be on top of a floating window is still not supported, or did I miss something. sorry I don't have the answer. But for a floating window to be automatically on top of other window( both tiling and floating ) when in focused the solution from [here](https://wiki.hyprland.org/Configuring/Binds/#multiple-binds-to-one-key) does work
```ini
# to switch between windows in a floating workspace
bind = SUPER, Tab, cyclenext,           # change focus to another window
bind = SUPER, Tab, bringactivetotop,    # bring it to the top
```

, but for a tiling window to be on top( on z-axis ) of floating windows when at least just in focused, please🙏 consider adding support for that. I've just posted this here due to regular users can't open an issue.

**Comment by johnwill97** [comment]
> Is there any way to achieve the desired behavior?

No, they don't support it. [#10619](https://github.com/hyprwm/Hyprland/discussions/10619)
